conf/modules/adc_expansion_uart.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="adc_expansion_uart">
+  <doc>
+    <description>Get analog signals from expansion board to main autopilot over uart</description>
```

Comment:
First line should be the name that is displayed in the generated list: http://docs.paparazziuav.org/latest/onboard_modules.html